src/components/document/document-or-browser.tsx

```diff
@@ -33,8 +34,10 @@ export const DocumentOrBrowser: React.FC<IDocumentOrBrowserProps> = props => {
 };
 
 function useTabSpec(tab: ENavTab) {
-  const appConfig = useAppConfigStore();
-  return appConfig.getNavTabSpec(tab);
+  const stores = useStores();
+  const navTabSpecs = getNavTabSpecsFromStores(stores);
+  const navTabSpec = navTabSpecs?.find( spec => spec.tab === tab );
```

Comment:
```suggestion
  const navTabSpec = navTabSpecs?.find(spec => spec.tab === tab);
```
Also, rather than having clients dig into the implementation details this way it seems cleaner to implement a `getNavTabSpec()` method on the `NavTabConfigModel`.